src/deepsparse/evaluation/utils.py

```diff
@@ -11,21 +11,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+import ast
+import logging
 import os
-from typing import Any, Dict, List, Optional, Tuple, Union
+from typing import Dict, List, Optional, Union
 
 from deepsparse import Pipeline
+from deepsparse.operators.engine_operator import DEEPSPARSE_ENGINE
 
 
 __all__ = [
     "create_pipeline",
     "get_save_path",
-    "args_to_dict",
+    "parse_kwarg_tuples",
     "resolve_integration",
 ]
-
-LM_EVALUATION_HARNESS = "lm-evaluation-harness"
+_LOGGER = logging.getLogger(__name__)
+LM_EVALUATION_HARNESS = "lm-eval-harness"
```

Comment:
Considering `lm-eval-harness` is still the name of the repo, I would propose to keep it as `lm-eval-harness`. I think if you'd like to alias `lm_eval` as well since that is the name of their CLI command, that would be fine

Reply:
Yes, this is something that has been introduced in this PR right? We are indeed committing to the name 'lm-eval-harness'

Reply:
sorry there are multiple typos - i meant to keep `lm-evaluation-harness` and contest the change to `lm-eval-harness`

Reply:
@mgoin note: this comes from the docs: https://neuralmagic.github.io/docs-v2/get-started/deploy (bottom of the page). I am happy to change this to whatever product sees fit.

Reply:
Let's get this in in the current state, I can always change this detail if needed.